.github/actions/publish-nuget/action.yml

```diff
@@ -200,26 +200,3 @@ runs:
           )}
 
         New-NuGetPackage @parameters
-
```

Comment:
This was removed only for testing so it won't publish to NuGet, if reviewed and approved, I will restore it.